File: javascript/7_kyu/testing_1_2_3.js
// Testing 1-2-3
// https://www.codewars.com/kata/54bf85e3d5b56c7a05000cf9/train/javascript

var number = function(array) {
    if ( array.length == 0 ) {
        return ans = [];
    } else {
        var seq = 1;
        var ans = [];

        for (var i = 0; i < array.length; i++) {
            ans.push(seq.toString() + ": " + array[i]);
            seq++;
        }
    }

    return ans
}

console.log(number(["a", "b", "c"]))

File: python/7_kyu/testing_1_2_3.py
# Testing 1-2-3
# https://www.codewars.com/kata/54bf85e3d5b56c7a05000cf9/train/python

def number(lines):
    seq, ans = 1, []

    for i in lines:
        ans.append(str(seq) + ": " + i)
        seq += 1
    
    return ans

print(number(["a", "b", "c"]))